src/styles/RangeDatePicker.scss

```diff
@@ -0,0 +1,52 @@
+.RangeDatePicker {
+
+    .react-datepicker-wrapper {
+        width: 100%;
+    }
+
+    .react-datepicker__day--in-range {
+        background: $setlife-blue;
+        &:hover {
+            background: $setlife-blue;
+        }
+    }
+    .react-datepicker__day--selected {
+        background: $setlife-blue;
+        &:hover {
+            background: $setlife-blue;
+        }
+    }
+    .react-datepicker__day--in-selecting-range {
+        background: $setlife-blue;
+        &:hover {
+            background: $setlife-blue;
+        }
+    }
+    .react-datepicker__day--keyboard-selected {
+        background: $setlife-blue;
```

Comment:
Just noticed a variable inconsistency `$setlife-blue` in kebab-case and `$light_grey` in snake_case so just making a note to capture this maybe in a code cleanup/refactor issue for the future